src/main/kotlin/org/amshove/kluent/Collections.kt

```diff
@@ -206,51 +206,51 @@ infix fun Short.shouldBeIn(theArray: ShortArray) = this.apply { this shouldBeIn
 
 infix fun Short.shouldNotBeIn(theArray: ShortArray) = this.apply { this shouldNotBeIn theArray.toTypedArray() }
 
-infix fun <T> Iterable<T>.shouldContain(theThing: T) = this.apply { if (this.contains(theThing)) Unit else fail("$this should contain $theThing", "$theThing", join(this)) }
+infix fun <T, I : Iterable<T>> I.shouldContain(theThing: T) = this.apply { if (this.contains(theThing)) Unit else fail("$this should contain $theThing", "$theThing", join(this)) }
```

Comment:
As last time, I think renaming the parameter to `expected` is more clear.

Reply:
You're right, I should do it in this PR, since I'm changing things there anyway